.env

```diff
@@ -1,3 +1,5 @@
 REACT_APP_CHAIN_ID="122"
 REACT_APP_NETWORK_URL="https://rpc.fuse.io/"
+REACT_APP_ROPSTEN_NETWORK_URL=https://ropsten.infura.io/v3/eab5c3b10b2b409e998a7528d7d81275
```

Comment:
that's your infura account, right?
can you please change to `2ba4786ec53f4b4aadd7ef61f37e5080`

Reply:
Yes, I will change that